### PR TITLE
Add `Rootstock` chain

### PR DESCRIPTION
**Pull request by ahsan-javaiid**
# Description

This PR adds rootstock chain to the list of supported networks in wallet-connect-live-app. 

About Rootstock
Website: https://rootstock.io/
Defillama: https://defillama.com/chain/Rootstock
Explorer: https://explorer.rootstock.io/
Developer portal: https://dev.rootstock.io/

# Context

We are working on to enable wallet connect via ledger for Rootstock in https://github.com/LedgerHQ/ledger-live. For that, purpose we need this PR to be merged so that we can add support for Rootstock in ledger-live for wallet connect. 

## Type of change

- [X] Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by ahsan-javaiid** [comment]
@Justkant Would be great if you can review and consider this pull request. Feel free to suggest if any changes required.

**Comment by ahsan-javaiid** [comment]
@Justkant We are working on to enable wallet connect for Rootstock in LedgerLive. This PR is required to be merged for this purpose. Let me know if you get a chance to review this pull request.

File: src/data/network.config.ts [hunk 1]
```diff
@@ -37,6 +37,13 @@ export const EIP155_CHAINS_MAINNET: Record<string, Network> = {
     displayName: "Polygon",
     color: "#6d29de",
   },
+  rootstock: {
```
Comment (Justkant) [inline src/data/network.config.ts:40]:
```suggestion
  rsk: {
```

File: localManifest.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
     "polygon",
     "bsc",
     "optimism",
+    "rootstock",
```
Comment (Justkant) [inline localManifest.json:23]:
```suggestion
    "rsk",
```

File: src/data/__tests__/networkConfig.test.ts [hunk 3]
```diff
@@ -20,6 +20,12 @@ describe("SUPPORTED_NETWORK", () => {
     expect(polygon?.displayName).toBe("Polygon");
   });
 
+  it("contains Rootstock", () => {
+    const rootstock: Network | undefined = SUPPORTED_NETWORK.rootstock;
```
Comment (Justkant) [inline src/data/__tests__/networkConfig.test.ts:24]:
```suggestion
    const rootstock: Network | undefined = SUPPORTED_NETWORK.rsk;
```

**Review by Justkant — CHANGES_REQUESTED** [review]
Thanks for the PR

I tested your PR and it was not working, the issue is only the currency name used in LL
Which is not `rootstock` but `rsk`, after apply my proposed changes everything seems to work correctly

We'll also have to update our manifest on the backend after merging this to make it work properly in prod

File: src/data/network.config.ts (hunk 1 above)
Comment (ahsan-javaiid) [reply src/data/network.config.ts]: Done!

File: src/data/__tests__/networkConfig.test.ts (hunk 3 above)
Comment (ahsan-javaiid) [reply src/data/__tests__/networkConfig.test.ts]: Done!

**Review by Justkant — APPROVED** [review]

**Pull request merged**